polyply/src/meta_molecule.py

```diff
@@ -285,10 +311,21 @@ def from_itp(cls, force_field, itp_file, mol_name):
     @classmethod
     def from_block(cls, force_field, mol_name):
         """
-        Constructs a :class::`MetaMolecule` from an vermouth.molecule.
+        Constructs a MetaMolecule from a vermouth.molecule.Block.
+
+        Parameters
+        ----------
+        force_field: :class:`vermouth.forcefield.ForceField`
+            the force-field that must contain the block
+        mol_name: str
+            name of the block matching a key in ForceField.blocks
+
+        Returns
+        -------
+        :class:`polyply.src.meta_molecule.MetaMolecule`
```

Comment:
```suggestion
        :class:`polyply.MetaMolecule`
```

Reply:
yeah we had some problems with sphinx that's why I have the habit of writing the full path. However, the problem has been resolved so this doc-string suggestion should be fine.